From a61f80817625502c7f6e47ea6730553ed56a2077 Mon Sep 17 00:00:00 2001
From: Arnd Bergmann <arnd@arndb.de>
Date: Mon, 17 Jun 2013 09:55:50 +0200
Subject: gpio: rcar: fix gpio_rcar_of_table

The device table needs to be terminated with an empty element.

Signed-off-by: Arnd Bergmann <arnd@arndb.de>
Cc: Laurent Pinchart <laurent.pinchart+renesas@ideasonboard.com>
Cc: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit 30d2266c685ce9f560e5023e4add58f890554a46)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 drivers/gpio/gpio-rcar.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpio/gpio-rcar.c b/drivers/gpio/gpio-rcar.c
index 34f86b26..e8198dd6 100644
--- a/drivers/gpio/gpio-rcar.c
+++ b/drivers/gpio/gpio-rcar.c
@@ -440,6 +440,7 @@ static const struct of_device_id gpio_rcar_of_table[] = {
 	{
 		.compatible = "renesas,gpio-rcar",
 	},
+	{ },
 };
 
 MODULE_DEVICE_TABLE(of, gpio_rcar_of_table);
-- 
1.8.4.3.gca3854a

